src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -1346,4 +1361,312 @@ class UiaTextRangeTests
             VERIFY_SUCCEEDED(utr->ScrollIntoView(alignToTop));
         }
     }
+
+    TEST_METHOD(GetAttributeValue)
+    {
+        Log::Comment(L"Check supported attributes");
+        IUnknown* notSupportedVal;
+        UiaGetReservedNotSupportedValue(&notSupportedVal);
+        for (long uiaAttributeId = 40000; uiaAttributeId <= 40042; ++uiaAttributeId)

```

Comment:
Consider adding a comment describing the meaning of these IDs.

Reply:
yes please!

Reply:
can you instead iterate over the constant values from UIA? `long x = UIA_FirstAttributeWeCareAbout; x <= UIA_LastAttributeWeCareAbout`?